From: Savvas Radevic <vicedar@gmail.com>
Description: Clementine is not listed in media applications 
 (right click > properties > open with) due to missing %U for 
 Exec in desktop file.
Bug: https://code.google.com/p/clementine-player/issues/detail?id=2848

Index: clementine-1.0.1+dfsg/dist/clementine.desktop
===================================================================
--- clementine-1.0.1+dfsg.orig/dist/clementine.desktop	2012-04-03 02:43:26.000000000 +0200
+++ clementine-1.0.1+dfsg/dist/clementine.desktop	2012-04-03 02:50:13.000000000 +0200
@@ -6,7 +6,7 @@
 GenericName[pt]=Reprodutor de músicas Clementine
 Comment=Plays music and last.fm streams
 Comment[pt]=Reprodução de músicas e emissões last.fm
-Exec=clementine
+Exec=clementine %U
 TryExec=clementine
 Icon=application-x-clementine
 Terminal=false
